### DISCUSSION TITLE
Several of your organizations have been flagged.

### DISCUSSION THREAD

**Discussion by pivotrootstech** [community/community#120045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am getting the message below all of sudden without any prior notification.Most of projects data is hidden and which seems to be available are not functional. Getting message **"Several of your organizations have been flagged.
Because of that, these organizations are hidden from the public. If you believe this is a mistake, please [contact support](https://support.github.com/contact/reinstatement) to have your organizations’ statuses reviewed."** Have a raised ticket for same but no resolution yet.Don't know the reason behind the cause of incident.Hope to get resolved at earliest.Can anyone suggest the fatest alternate way to resolve the issue. Thanks

**Comment by michael-pisman** [comment]
Just had my org flagged as well. No idea why, any one has an insight into what can cause an org to get flagged?

**Reply by louanfontenele** [reply]
The same thing happened to me, and the worst part is that it happens without explanation; you don't even know what to do to fix it!

**Comment by pivotrootstech** [comment]
Support team is yet to revert on the tickets.
Atleast they should revert reason for such action
https://support.github.com/ticket/personal/0/2766766
